qa/tasks/cephfs/test_misc.py

```diff
@@ -912,3 +912,50 @@ def test_no_fs_id_incr_on_fs_creation_fail(self):
                 test_fs.destroy()
             if test_fs1 is not None:
                 test_fs1.destroy()
+
+class TestForwardAuth(CephFSTestCase):
+    MDSS_REQUIRED = 2
+    CLIENTS_REQUIRED = 1
+
+    def test_client_request_forward_to_auth_mds(self):
+        """
+        The WRITE client requests will always be forwarded to the auth MDSs.
+        """
+        self.fs.set_max_mds(2)
```

Comment:
```suggestion
        if not isinstance(self.mount_a, FuseMount):
            self.skipTest("<fillin>")
        self.fs.set_max_mds(2)
```

Reply:
Already done this in the `postmerge`, but I will fix it here anyway.

Reply:
I understand but I think it's important for that requirement to be codified in the test as well.

Reply:
> I understand but I think it's important for that requirement to be codified in the test as well.

Sure, done.